server/services/v1/query_runner.go

```diff
@@ -223,7 +223,7 @@ func (runner *BaseQueryRunner) insertOrReplace(ctx context.Context, tx transacti
 	allKeys := make([][]byte, 0, len(documents))
 	for _, doc := range documents {
 		// reset it back to doc
-		doc, err = runner.mutateAndValidatePayload(coll, doc)
+		doc, err = runner.mutateAndValidatePayload(coll, ts.ToRFC3339(), "", doc)
```

Comment:
As I mentioned earlier I think this shouldn't be a separate `createdAt` and `updatedAt` timestamp just one timestamp value to be used for the mutation of this document.

Reply:
But are you suggesting not to support @createdAt/@updatedAt and just support something like @timestamp that is updated always either during insert/update?

Reply:
We discussed about it.